.gitignore

```diff
@@ -19,5 +19,12 @@ venv/
 *.sublime-*
 MCprep_addon/import_bridge/conf
 MCprep_addon/import_bridge/nbt
-MCprep_addon/MCprep_resources/
+MCprep_addon/MCprep_resources/*.blend
```

Comment:
Before this change, we were masking .po files, felt it was worth making sure we had it recognized explicitly.